documentation/modules/ROOT/pages/tutorial.adoc

```diff
@@ -9,7 +9,7 @@ include::_attributes.adoc[]
 
 toc::[]
 
-This tutorial walks you through running Debezium {debezium-version} for change data capture (CDC). You will use Docker (1.9 or later) to start the Debezium services, run a MySQL database server with a simple example database, use Debezium to monitor the database, and see the resulting event streams respond as the data in the database changes.
```

Comment:
Hum, hum. So this one actually could remain, as it's not a temporary info such as "As of ...", but just describes the current version of the tutorial? WDYT?

Reply:
I guess my train of thought here was that the version selected in the documentation clarifies the version of the tutorial being viewed and placing it in the text here felt redundant; however if you prefer it to remain I can revert this one.  I can see it both ways.

Reply:
Yes, I think I'd prefer to add it back. It's a nice confirmation to make sure one is reading the right version.

Reply:
Reverted.